app/views/layouts/frontend.html.erb

```diff
@@ -1,13 +1,77 @@
-    <ul id="proposition-links">
-      <li><%= main_navigation_link_to "Departments", organisations_path %></li>
-      <li><%= main_navigation_link_to "Worldwide", world_locations_path(locale: :en) %></li>
-      <li><%= main_navigation_link_to 'How government works', how_government_works_path %></li>
-      <li><%= main_navigation_link_to "Get involved", get_involved_path %></li>
-      <li class="clear-child"><a href="<%= CGI::escapeHTML('/search/policy-papers-and-consultations?content_store_document_type[]=open_consultations&content_store_document_type[]=closed_consultations') %>">Consultations</a></li>
-      <li><a href="/search/research-and-statistics">Statistics</a></li>
-      <li><a href="/news-and-communications">News and communications</a></li>
+    <ul id="proposition-links" data-module="gem-track-click">
+      <li>
+        <%= main_navigation_link_to "Departments",
+            organisations_path,
+            data: {
+              track_category: "headerClicked",
+              track_action: "departmentsLink",
+              track_label: organisations_path,
+              track_dimension: "Departments",
+              track_dimension_index: "29",
+            }
+        %>
+      </li>
+      <li>
+        <%= main_navigation_link_to "Worldwide",
+            world_locations_path(locale: :en),
+            data: {
+              track_category: "headerClicked",
+              track_action: "worldwideLink",
+              track_label: world_locations_path(locale: :en),
+              track_dimension: "Worldwide",
+              track_dimension_index: "29",
+            }
+        %>
+      </li>
+      <li>
+        <%= main_navigation_link_to "How government works",
+            how_government_works_path,
+            data: {
+              track_category: "headerClicked",
+              track_action: "governmentactivityLink",
+              track_label: how_government_works_path,
+              track_dimension: "How government works",
+              track_dimension_index: "29",
+            }
+        %>
+      </li>
+      <li>
+        <%= main_navigation_link_to "Get involved",
+            get_involved_path,
+            data: {
+              track_category: "headerClicked",
+              track_action: "governmentactivityLink",
+              track_label: get_involved_path,
+              track_dimension: "Get involved",
+              track_dimension_index: "29",
+            }
+        %>
+      </li>
+      <% consultations_path = CGI::escapeHTML('/search/policy-papers-and-consultations?content_store_document_type[]=open_consultations&content_store_document_type[]=closed_consultations') %>
+      <li class="clear-child"
+          data-track-category="headerClicked"
+          data-track-action="governmentactivityLink"
+          data-track-label="<%= consultations_path %>"
+          data-track-dimension="Consultations"
+          data-track-dimension-index="29">
```

Comment:
Just wanted to check - should these data attributes being on the list item or the link?

Reply:
🤦

Reply:
I did gooder now @injms